src/preload-hover.js

```diff
@@ -12,20 +12,23 @@ export default class PreloadHover {
   }
 
   start(domScope = this.configuration.defaultDomScope) {
-    if (!domScope) { throw new Error('domScope must be provided.'); }
+    if (!domScope) { throw new Error('domScope must be provided.'); } 
+    domScope = [...new Set(domScope)];
+    domScope.forEach(domScope => {
+      const head = document.getElementsByTagName('head')[0];
```

Comment:
`head` can be moved outside the `forEach` since it should be the same for the entire page.